assignments/objects.js

```diff
@@ -18,28 +18,79 @@ const example = {
 }
 
 // Write your intern objects here:
+const intern1 = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const intern2 = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
+
+const intern3 = {
+  "id": 3,
+  "name": "Kevin",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "F"
+}
+
+const intern4 = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+
+const intern5 = {
+  "id": 5,
+  "name": "Antoinnette",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+}
 
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.log(intern1.name);
 
 // Kennan's ID
+console.log(intern2.id);
 
 // Keven's email
+console.log(intern3.email);
 
 // Gannie's name
+console.log(intern4.name);
 
 // Antonietta's Gender
+console.log(intern5.gender);
+
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+intern2.speak = function () {
+  console.log(`Hello, my name is ${this.name}.`);
+};
+intern2.speak();
+
+
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
 
+antonietta.multiplyNums = function(num1, num2) {
+  return num1 * num2;
+};
+
+console.log(antonietta.multiplyNums(3,4));
```

Comment:
i think we discussed this last night, but since you have not defined a variable named 'antonietta' this code will error out. it should be pointed at intern5

```
intern5.multiplyNums = function(num1, num2) {
  return num1 * num2;
};
console.log(intern5.multiplyNums(3,4));
```